### ISSUE TITLE
Crash at the start of version 0.2.2

### ISSUE THREAD

**Issue by famellad** [CrankshaftForSteam/crankshaft#9]
Crankshaft just crashes when I open Steam, this is the relevant output:

```
2022/08/12 15:40:46 steamwebhelper found!
2022/08/12 15:40:46 Waiting to connect to Steam client...
2022/08/12 15:40:59 Connected!
2022/08/12 15:41:00 Patching /home/famellad/.steam/steam/steamui/libraryroot~sp.js...
2022/08/12 15:41:00 Error: open /home/famellad/.steam/steam/steamui/libraryroot~sp.js: no such file or directory
```

I am using Arch Linux, and I have installed Steam through the repositories. The file Crankshaft claims doesn't exist does not, in fact, exist, so I'm not sure what the deal is.

Anyway, I will happily provide any additional information required. Sorry I can't help more, but I don't really understand how Crankshaft actually works :(

**Comment by danyi** [comment]
I seem to have the same problem, mine is Steamos

**Comment by danyi** [comment]
[danyi@holosteamos anime-game-launcher]$ flatpak run space.crankshaft.Crankshaft
2022/08/18 20:52:37 Starting system tray icon...

(process:2): libappindicator-CRITICAL **: 20:52:37.576: app_indicator_set_title: assertion 'IS_APP_INDICATOR (self)' failed

(process:2): libappindicator-CRITICAL **: 20:52:37.577: app_indicator_set_label: assertion 'IS_APP_INDICATOR (self)' failed
Gtk-Message: 20:52:37.577: Failed to load module "canberra-gtk-module"
2022/08/18 20:52:37 Listening on :8085
2022/08/18 20:52:37 Waiting to connect to Steam client...
2022/08/18 20:52:38 Connected!
2022/08/18 20:52:38 Patching /home/danyi/.steam/steam/steamui/libraryroot~sp.js...
2022/08/18 20:52:38 Copying original /home/danyi/.steam/steam/steamui/libraryroot~sp.js to /home/danyi/.steam/steam/steamui/libraryroot~sp.orig.js...
2022/08/18 20:52:38 Using cached file
2022/08/18 20:52:38 Patching /home/danyi/.steam/steam/steamui/sp.js...
2022/08/18 20:52:38 Copying original /home/danyi/.steam/steam/steamui/sp.js to /home/danyi/.steam/steam/steamui/sp.orig.js...
2022/08/18 20:52:38 Unminifying...
2022/08/18 20:52:38 /app/bin/js-beautify -o /home/danyi/.steam/steam/steamui/sp.unmin.js /home/danyi/.steam/steam/steamui/sp.js
2022/08/18 20:52:39 Patching main menu...
2022/08/18 20:52:39 React not found
2022/08/18 20:52:58 Force reload from systray
2022/08/18 20:52:58 Waiting to connect to Steam client...
2022/08/18 20:52:59 Connected!
2022/08/18 20:53:00 Patching /home/danyi/.steam/steam/steamui/libraryroot~sp.js...
2022/08/18 20:53:00 Copying original /home/danyi/.steam/steam/steamui/libraryroot~sp.js to /home/danyi/.steam/steam/steamui/libraryroot~sp.orig.js...
2022/08/18 20:53:00 Using cached file
2022/08/18 20:53:00 Patching /home/danyi/.steam/steam/steamui/sp.js...
2022/08/18 20:53:00 Copying original /home/danyi/.steam/steam/steamui/sp.js to /home/danyi/.steam/steam/steamui/sp.orig.js...
2022/08/18 20:53:00 Unminifying...
2022/08/18 20:53:00 /app/bin/js-beautify -o /home/danyi/.steam/steam/steamui/sp.unmin.js /home/danyi/.steam/steam/steamui/sp.js
2022/08/18 20:53:00 Patching main menu...
2022/08/18 20:53:00 React not found
2022/08/18 20:56:32 Cleaning up patched scripts before exiting
2022/08/18 20:56:32 Writing original libraryroot~sp.js
2022/08/18 20:56:32 Writing original sp.js

**Comment by danyi** [comment]
You're right. After I switched to the stable version, it can be used normally 

 Thank you！！！！！！！！
